examples/imgui_impl_sdl.cpp

```diff
@@ -123,6 +125,12 @@ static bool ImGui_ImplSDL2_Init(SDL_Window* window)
 {
     g_Window = window;
 
+    // check and store if we are on Wayland
+    {
+        std::string currentVideoDriver(SDL_GetCurrentVideoDriver());
+        g_SDL_VIDEODRIVER_IS_WAYLAND = currentVideoDriver == "wayland";
+    }
+
```

Comment:
Couldn't wayland be figured out at compile time via a define?

Reply:
unfortunately no, the SDL_VIDEODRIVER environment variable is read during runtime